config/hotreloadable.go

```diff
@@ -1,14 +1,39 @@
 package config
 
-import "time"
+import (
+	"sync/atomic"
+	"time"
+)
 
 // RegisterIntConfigVariable registers int config variable
 func RegisterIntConfigVariable(defaultValue int, ptr *int, isHotReloadable bool, valueScale int, keys ...string) {
```

Comment:
[optional] we could deprecate this one as well and create two alias functions or not deprecate anything at all, not even `*Config.RegisterIntConfigVariable`.